martz_cohort4_css_positioning_you_got_this.html

```diff
@@ -138,10 +343,10 @@
               <!-- card content -->
               <div class="card-body">
                 <div class="media d-block d-md-flex mt-3">
-                  <img class="d-flex mb-3 mx-auto z-depth-1" src="img/jennie-brown-MKBoRZEGeiM-unsplash.jpg"
-                    alt="placeholder image" style="width: 100px;">
+                  <img class="d-flex mb-3 mx-auto z-depth-1" src="img/Andrea-prof.jpg"
+                    alt="photo of the author" style="width: 100px;border-radius:50%;">
                   <div class="media-body text-center text-md-left ml-md-3 ml-0">
-                    <h5 class="mt-0 font-weight-bold">Name of Author</h5>
+                    <h5 class="mt-0 font-weight-bold">Andrea Martz</h5>
                     Dragée biscuit jelly-o tootsie roll cake chupa chups. Cupcake fruitcake bear claw danish marzipan
                     gummi
                     bears halvah chocolate cake lemon drops. Icing I love icing caramels. Toffee soufflé croissant
```

Comment:
I think you meant to add your about me section here? Thanks!

Reply:
Thanks Dana.  Could you cancel this pull request?  My local copy shows an About Me section.  I'll resubmit the pull request with everything updated.